DQM/TrackingMonitor/src/TrackAnalyzer.cc

```diff
@@ -977,6 +987,7 @@ void TrackAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSe
     const reco::BeamSpot& bs = *recoBeamSpotHandle;
 
     DistanceOfClosestApproachToBS      -> Fill(track.dxy(bs.position()));
+    AbsDistanceOfClosestApproachToBS   -> Fill(abs(track.dxy(bs.position())));
```

Comment:
indeed, look a bit binned at the moment
![screen shot 2016-06-10 at 12 54 14 pm](https://cloud.githubusercontent.com/assets/4143702/15962305/758a0548-2f0a-11e6-8a38-2f4e01cbdffb.png)